mappings/net/minecraft/block/Block.mapping

```diff
@@ -1,11 +1,11 @@
 CLASS none/akw net/minecraft/block/Block
 	FIELD A stateFactory Lnone/ask;
-	FIELD a id Lnone/kp;
+	FIELD a ID Lnone/kp;
 	FIELD b creativeTab Lnone/adq;
 	FIELD c defaultState Lnone/asj;
 	FIELD d unlocalizedName Ljava/lang/String;
-	FIELD h registry Lnone/cs;
-	FIELD i blockstateList Lnone/cx;
+	FIELD h REGISTRY Lnone/cs;
+	FIELD i BLOCKSTATELIST Lnone/cx;
```

Comment:
I had seen these were not named the same as other static final things like the item registry.